ci/Jenkinsfile_utils.groovy

```diff
@@ -186,18 +186,27 @@ def update_github_commit_status(state, message) {
     context = get_github_context()
     echo "context=${context}"
 
-    step([
-      $class: 'GitHubCommitStatusSetter',
-      reposSource: [$class: "ManuallyEnteredRepositorySource", url: repoUrl],
-      contextSource: [$class: "ManuallyEnteredCommitContextSource", context: context],
-      commitShaSource: [$class: "ManuallyEnteredShaSource", sha: commitSha],
-      statusBackrefSource: [$class: "ManuallyEnteredBackrefSource", backref: "${env.RUN_DISPLAY_URL}"],
-      errorHandlers: [[$class: 'ShallowAnyErrorHandler']],
-      statusResultSource: [
-        $class: 'ConditionalStatusResultSource',
-        results: [[$class: "AnyBuildResult", message: message, state: state]]
-      ]
-    ])
+    // a few attempts need to be made: https://github.com/apache/incubator-mxnet/issues/11654
+    for (int attempt = 1; attempt <= 3; attempt++) {
```

Comment:
It seemed not to be an attempt, it will run three times no matter what if the first time succeeded or not. Please correct me if I understand wrongly.

Reply:
@lanking520 yes, you understand correctly. Do you suggest to rename the variable?

Reply:
@lebeg Please check the attempt if it successful by checking the result (status:200).

Reply:
@lanking520 the return code of what you suggest to check?

Reply:
@lanking520 the problem is that this call doesn't return a value to check against. The strategy here is really to just hit the github endpoint three times (instead of once) as a mitigation for the github status problem. We realise it's not elegant, and that it doesn't _solve_ the problem. But this is just a stop-gap measure to improve the the situation for the developers until we can finally fix the issue.

Reply:
Maybe I'll mention again here the reasons mentioned in the description of this PR for clarity:
> * Using a different Jenkins plugin GitHub Integration Plugin has been considered, but unfortunately after installation there was not way to configure it - the options were missing from the configuration screen
> * All other plugins required explicit credentials passing as arguments
> * No way of asking for the commit status for verification has been found to check the status after the update
> * No way for checking the list of repositories that need to be updated was found (and whether it's empty) - all what happens, happens internally in the plugin